### DISCUSSION TITLE
Position of currency symbol

### DISCUSSION THREAD

**Discussion by Blackfaded** [dm4t2/vue-currency-input#340] [Q&A] [answered]
Hey there,

first I want to thank you for the great work. Really nice library!

Is there a possibility to add an option for the currency symbols position?
So that I can place it like this:
![Bildschirmfoto 2022-08-24 um 14 53 28](https://user-images.githubusercontent.com/19570581/186423706-431d3d6f-3856-434b-a64c-73d159124fe2.png)

Greetings!

**Comment by dm4t2** [comment, marked as answer]
Hi, you can set the [`currencyDisplay` option](https://dm4t2.github.io/vue-currency-input/config.html#currencydisplay) to "hidden" and then place the currency symbol in a content slot of your input component.
For example with Quasar: https://codesandbox.io/s/using-vue-currency-input-with-quasar-forked-87zq2w
